os/tools/symtab_formatter.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+###########################################################################
+#
+# Copyright 2018 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+# File 	 : symtab_formatter.sh
+
+# Compile symbol table formatter application
+gcc tools/symtab_formatter.c -o symtab
+
+#execute
+./symtab
+
+# clean up
+rm -f symtab
+
+echo "Succesfully generated Symtab.bin file"
```

Comment:
Please modify the spell mistakes
Succesfully ==> Successfully

Reply:
okay